css/bulma.css

```diff
@@ -5997,7 +5979,7 @@ body.has-navbar-fixed-bottom {
 }
 
 .navbar-burger:hover {
-  background-color: rgba(0, 0, 0, 0.05);
```

Comment:
This was a hardcoded black and was changed to scheme-invert for themeability reasons